docs/readthedocs/source/doc/PPML/QuickStart/deploy_intel_sgx_device_plugin_for_kubernets.md

```diff
@@ -0,0 +1,26 @@
+# Deploy the Intel SGX Device Plugin for Kubenetes
+
+The instructions in this section are modified from the [Intel SGX Device Plugin homepage][intelSGX], to which please refer should questions arise.
+
+## Prerequisites
+Prerequisites for building and running these device plugins include:
+- Appropriate hardware
```

Comment:
Add after hardware. [3rd Gen Intel Xeon Scalable Processors](https://www.intel.com/content/www/us/en/products/docs/processors/xeon/3rd-gen-xeon-scalable-processors-brief.html)

This change can help customer without SGX platform knowledge